src/layout/Section.jsx

```diff
@@ -47,6 +51,10 @@ Section.propTypes = {
 	noSeparator: PropTypes.oneOfType([
 		PropTypes.bool,
 		PropTypes.oneOf(Object.keys(VALID_BREAKPOINTS))
+	]),
+	flushUntil: PropTypes.oneOfType([
+		PropTypes.bool,
```

Comment:
Makes sense to me. I wanted to add a way to _always_ flush a `<Section>`, but I guess it's rare enough that we could just do it in `sasstools`